controllers/storagecluster/noobaa_system_reconciler.go

```diff
@@ -240,6 +241,9 @@ func (r *StorageClusterReconciler) setNooBaaDesiredState(nb *nbv1.NooBaa, sc *oc
 		}
 	}
 
+	isEnabled, rotationSchedule := util.GetKeyRotationSpec(sc)
+	nb.Spec.Security.KeyManagementService.EnableKeyRotation = isEnabled
+	nb.Spec.Security.KeyManagementService.Schedule = rotationSchedule
```

Comment:
looks like NooBaa is always looking for the spec under KMS: https://github.com/noobaa/noobaa-operator/blob/master/pkg/apis/noobaa/v1alpha1/noobaa_types.go#L254